examples/common_setup.sh

```diff
@@ -3,9 +3,9 @@
 EXAMPLES=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
 
 # Use this for tests or things you're not quite sure of
-root_coll="u/${USER}/cm"
+#root_coll="u/${USER}/cm"
 # Use this for runs you want in the butler in the traditional place
-#root_coll="HSC/runs/RC2"
+root_coll="HSC/runs/RC2"
```

Comment:
You probably want to leave the default as the generic test root_coll of  as "u/${USER}/cm".

Reply:
Oop, I didn't think this would get pulled in. I can clean up the pull request later today or before this gets merged, thanks.